docs/src/main/asciidoc/rest-json.adoc

```diff
@@ -181,58 +184,58 @@ It will allow to narrow down the number of JAX-RS providers (which can be seen a
 
 === Configuring JSON support
 
-==== JSON-B
+==== Jackson
 
-Quarkus makes it very easy to configure various JSON-B settings via CDI beans. The simplest (and suggested) approach is to define a CDI bean of type `io.quarkus.jsonb.JsonbConfigCustomizer`
-inside of which any JSON-B configuration can be applied.
+As stated above, Quarkus provides the option of using https://github.com/FasterXML/jackson[Jackson] instead of JSON-B via the use of the `quarkus-resteasy-jackson` extension.
```

Comment:
Nitpicking: "Jackson instead of JSON-B" still reads a little like JSON-B is preferred.

Reply:
Ah ah, good catch. I just moved the sections, didn't think they would need changes.

I fixed it.